.github/workflows/release.yml

```diff
@@ -1,45 +1,99 @@
 name: Release Packages
 
 on:
-  workflow_dispatch:
-    inputs:
-      libraries:
-        description: 'List of libraries to release'
-        default: 'meta.libraries = []'
-        type: string
-        required: false
```

Comment:
Are we using semantic release to control what this workflow us releasing?

Reply:
Semantic Release handles just pushing to pypi. I haven't done the code for what packs will be updated or not but it won't be an issue since the zips would match exactly whats on the release branch. this could be a future optimization.

Reply:
Thats gonna be important since currently, running a release will push out any unfinished packs assuming they compile.

Reply:
Yea, but it won't actually trigger a change in git bc it *should* be a 1:1 match with the zip on the branch.